### PR TITLE
Init Gradle

### PR DESCRIPTION
**Pull request by Oleh-Stefanovych**
Changes in this PR:
- extract UI implementation into a separate module;
- add [Gradle 5.6.2](https://docs.gradle.org/5.6.2/release-notes.html) configuration;
- add `server` java module;
- minor changes in `.gitignore`.

### CODE REVIEW

**Comment by Oleh-Stefanovych** [comment]
@okucheruk, PTAL.

**Comment by Oleh-Stefanovych** [comment]
@denysLavrienov, @kuzmavictor, PTAL.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+.gradle
+
+# IntelliJ IDEA build files
+out/
+
+# Gradle build files
+build/
+
+# Node.js build fiels
+web-client/node_modules
```
Comment (okucheruk) [inline .gitignore]: Let's have a consistency - ignoring folders should end with `/`.
Please fix or all cases.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+# Gradle interim configs
+.gradle
+
+# IntelliJ IDEA build files
+out/
+
+# Gradle build files
+build/
+
+# Node.js build fiels
```
[the remark this answers is not in the document]
Comment (Oleh-Stefanovych) [reply .gitignore]: Let's discuss in person

**Comment by Oleh-Stefanovych** [comment]
@okucheruk, fixed, PTAL.

Fixies:
- Add slash to ignoring folders path in `.gitignore`.
- Fix documentation in `.gitignore`.

**Comment by Oleh-Stefanovych** [comment]
@armiol, PTAL.

**Pull request merged**